lib/accesscontrol/permitted_action_on_object_query.rb

```diff
@@ -31,6 +31,30 @@ def can?(action_id, object_type, object_id)
       end
     end
 
+    # Allow permission on an ActiveRecord object.
+    # A grant is universally unique and is enforced at the database level.
+    #
+    # @param action_id [Integer] The action to grant for the object
+    # @param object_type [ActiveRecord::Base] The ActiveRecord model that receives a permission grant.
+    # @param object_id [Integer] The id of the ActiveRecord object which receives a permission grant
+    # @return [nil] Returns nil if successful, otherwise will raise an Error (AccessControl::CouldNotGrantError).
+    #
+    # @example
+    #   # Allow the user access to Post 7
+    #   AccessControl::Query.new(user).grant(3, Post, 7)
+    def grant(action_id, object_type, object_id)
+      a = PermittedActionOnObject.create(
```

Comment:
Is there a need for the local variable `a`? If so, can we name it something more clear? If not, remove it?

Reply:
damn - that shouldn't be there